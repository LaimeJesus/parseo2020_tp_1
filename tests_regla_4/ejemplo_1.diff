1c1
< Hay variables en signatura repetidas
\ No newline at end of file
---
> Hay variables en signatura repetidas
