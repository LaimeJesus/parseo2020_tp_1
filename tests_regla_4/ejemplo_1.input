fun f : x , y -> y