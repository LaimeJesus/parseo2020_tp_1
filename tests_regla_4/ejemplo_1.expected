Hay variables en signatura repetidas